extensions/wikia/AdEngine/ResourceLoaders/ResourceLoaderAdEngineBase.php

```diff
@@ -23,7 +23,7 @@
 	 * @return bool|MWHttpRequest|string
 	 */
 	protected function fetchRemoteScript( $url ) {
-		return Http::get( $url );
+		return Http::get( $url, 'default', [ 'noProxy' => true ] );
```

Comment:
LGTM.

As we discussed earlier, let's debug the root cause tomorrow morning :)